### PR TITLE
Check for GMP module

### PR DESCRIPTION
**Pull request by annando**
See https://github.com/friendica/friendica/issues/11631#issuecomment-1186152231

### CODE REVIEW

**Comment by tobiasd** [comment]
Please update the installation documentation accordingly if this is now a hard dependency to the server.

Also, the messages.po file needs to be regenerated as you've added new strings for the translation.

File: src/Contact/Avatar.php [hunk 1]
```diff
@@ -263,7 +263,7 @@ private static function deleteCacheFile(string $avatar)
 	{
 		$localFile = self::getCacheFile($avatar);
 		if (!empty($localFile)) {
-			unlink($localFile);
+			@unlink($localFile);
```
Comment (nupplaphil) [inline src/Contact/Avatar.php:266]: I'd catch such exceptions instead of suppressing it

**Review by nupplaphil — CHANGES_REQUESTED** [review]

File: src/Contact/Avatar.php (hunk 1 above)
Comment (annando) [reply src/Contact/Avatar.php:266]: This is just to oppress this message that can occur during deletion when the file had been deleted via some other process after their existence had been checked in this process. There is nothing to handle nor anything to log here. So we could blow it up in some `try` ... `except` ... thingy without anything in the `except` part - or simply that `@`.

**Comment by MrPetovan** [comment]
Is this an optional dependency? For what feature?

**Comment by annando** [comment]
This is used by some part of the API that performs push notifications: https://github.com/friendica/friendica/blob/develop/src/Worker/PushSubscription.php

This is not new, but by now we hadn't checked for this module.

File: src/Worker/PushSubscription.php [hunk 2]
```diff
@@ -102,7 +102,6 @@ public static function execute(int $sid, int $nid)
 			],
 		]);
 
-		// @todo Only used for logging?
```
Comment (Quix0r) [inline src/Worker/PushSubscription.php:105]: Ah, okay. I found the line not only using `$payload` for logging.

**Review by Quix0r — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**